server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/Worker.scala

```diff
@@ -90,6 +90,7 @@ private[deploy] class Worker(
   memoryTracker.registerMemoryListener(storageManager)
 
   val partitionsSorter = new PartitionFilesSorter(memoryTracker, conf, workerSource)
+  partitionsSorter.setStorageManager(storageManager)
```

Comment:
Why not `val partitionsSorter = new PartitionFilesSorter(memoryTracker, conf, workerSource, storageManager)`

Reply:
I want to change the code as less as it can be.